.github/workflows/Umbrella_Sandbox_Setup.yml

```
@@ -0,0 +1,107 @@
+###############################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+name: Umbrella Sandbox Setup
+
+on:
+  #for testing purpouses
+  pull_request:
+    branches:
+      - 'main'
```

we should restrict the triggering of the workflow to only when really needed (it runs about 6 minutes or longer and otherwise just waste resources)

```suggestion
    branches:
      - 'main'
    paths:
      - 'src/**'
```